frontend/src/app/admin/permit-application-list/permit-application-list.component.html

```diff
@@ -0,0 +1,46 @@
+<section class="usa-section">
+  <div class="usa-grid">
+    <h1>Mount Baker-Snoqualmie National Forest</h1>
+    <h2>Non-commercial group use applications</h2>
+    <div *ngIf="apiErrors" class="usa-alert usa-alert-error" role="alert">
+      <div class="usa-alert-body">
+        <h3 class="usa-alert-heading">There are errors preventing your data from loading</h3>
+        <ul class="usa-alert-text">
+          <li *ngFor="let error of apiErrors">{{error}}</li>
+        </ul>
+      </div>
+    </div>
+    <div class="application-list">
+      <div class="application-field-list-labels">
+        <div>&nbsp;</div>
+        <div>Event name</div>
+        <div>Date submitted</div>
+        <div>Start date</div>
+        <div>End date</div>
+        <div>Location</div>
+        <div>Permit holder</div>
+        <div>&nbsp;</div>
+      </div>
+      <div *ngFor="let application of applications" [hidden]="application.status !== 'Received' && application.status !== 'Hold'" class="application-field-list-values">
+        <div *ngIf="application.status === 'Hold'" class="on-hold"><img src="/assets/img/on-hold-icon.svg"><span>On hold</span></div>
+        <div *ngIf="application.status === 'Received'">&nbsp;</div>
+        <!-- TODO: remove these debugging status indicators -->
+        <div *ngIf="application.status === 'Accepted'">👍</div>
+        <div *ngIf="application.status === 'Returned'">👎</div>
```

Comment:
They are indeed cute, but emojis are not particularly accessible or clear. I didn't notice these in the front end -- are they just appearing here?